packages/async-loader/async.js

```diff
@@ -15,15 +15,20 @@ export default function(req) {
 		this.shouldComponentUpdate = (_, nxt) => {
 			nxt = nxt.child === void 0;
 			if (nxt && old === void 0 && b) {
-				old = b.nodeType===3 ? b.data : h(b.nodeName, { dangerouslySetInnerHTML: { __html: b.innerHTML } });
+				old =
+					b.nodeType === 3
+						? b.data
+						: h(b.nodeName, {
```

Comment:
^ This is one of the things I dislike about Prettier. 
I don't think anyone would argue this is easier to read :P

Reply:
isnt there any setting to turn this off?